### ISSUE TITLE
Developer FAQ - I want to make a change to the schema and use it in the PDP

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction-docs#339]

**Comment by prinzdezibel** [comment]
@zenweasel  Does it make sense to use existing beesknees example plugin for this?

**Comment by zenweasel** [comment]
I feel like this is probably going to change pretty soon with the work that @aldeed is doing so we actually may want to wait on this one

**Comment by prinzdezibel** [comment]
@zenweasel Actually already done. But the extension for the schema is only one part of it.

The idea was to extend the product schema with coordinates to be able to show the user, where this product can be bought (e.g. in which store)

**Comment by zenweasel** [comment]
What is already done?

**Comment by prinzdezibel** [comment]
Please see:
https://github.com/reactioncommerce/reaction-example-plugin/pull/22/

I would need to write documentation for it, which I could to tomorrow.

**Comment by prinzdezibel** [comment]
And this is the PR for reaction documentation: https://github.com/reactioncommerce/reaction-docs/pull/358

**Comment by prinzdezibel** [comment]
@zenweasel If this PR gets approved we may link the article (which is now referred to as yet another add-on to the base tutorial on this page: https://docs.reactioncommerce.com/reaction-docs/master/plugin-complete-10) from our FAQ section again (so we have it linked twice).

**Comment by prinzdezibel** [comment]
Done: https://docs.reactioncommerce.com/reaction-docs/master/faqs
